components/Players/AudioPlayer.vue

```diff
@@ -2,16 +2,17 @@
   <audio
     ref="audioPlayer"
     autoplay
-    @timeupdate="onAudioProgress"
+    @timeupdate="throttledOnAudioProgress"
     @pause="onAudioPause"
-    @play="onPlay"
+    @play="unpause"
```

Comment:
```suggestion
    @play="onPlay"
```

As discussed [here](https://matrix.to/#/!flDUDlxzYhNVyLEcSA:bonifacelabs.ca/$HSWHNictJYCEaOyNqRF6tbX57JqPxSA53kOMYibIcDo?via=matrix.org&via=matrix.possumlodge.me&via=bonifacelabs.ca), let's stick to the conventional names for event callbacks: ``onX``

Reply:
Done